src/App.tsx

```diff
@@ -44,6 +45,7 @@ function App(isSignedIn: AppProps) {
       <AuthenticatedRoute path="/customers/customer/meter-readings/create" component={AddMeterReading} exact />
       <AuthenticatedRoute path={'/customers/customer/records'} component={CustomerRecords} exact />
       <AuthenticatedRoute path="/financial-summaries" component={FinancialSummariesMain} />
+      <AuthenticatedRoute path="/confirm-payment" component={ConfirmPayment} />
```

Comment:
We'll want to change this route once we've built out the remainder of the Financial Summary screens. It'll eventually look something like `/financial-summaries/financial-summary/payment/create` imo. Can we add a TODO above this to change the route name once the other screens are built out?